evaluate_model.py

```diff
@@ -278,6 +304,12 @@ def evaluate_model(transcribe_fn, ds, text_column):
         ds = datasets.load_dataset(dataset_name)[dataset_split]
 
     print(f"Beginning evaluation.")
-    res = evaluate_model(transcribe_fn, ds, ds_text_column)
+    metrics, results_df = evaluate_model(transcribe_fn, ds, ds_text_column)
+
+    print(f"Evaluation done. WER={metrics.wer}, WIL={metrics.wil}.")
+    
+    file_name = f"{args.model.replace('/', '_')}_{dataset_name.replace('/', '_')}_results.csv"
```

Comment:
How about have that as an argument to the script, and also separately add the model as a column?
That way the user can decide on calling conventions themselves.